src/routes/files.test.js

```diff
@@ -72,6 +78,28 @@ describe('Forms route', () => {
         message: 'Found'
       })
     })
+
+    test('Testing POST /file/link route returns an S3 link', async () => {
+      jest
+        .mocked(getPresignedLink)
+        .mockResolvedValue('https://s3.dummy.com/file.txt')
+
+      const response = await server.inject({
+        method: 'POST',
+        url: '/file/link',
+        auth,
+        payload: {
+          fileId: '1234',
+          retrievalKey: 'test'
+        }
+      })
+
+      expect(response.statusCode).toEqual(StatusCodes.OK)
+      expect(response.headers['content-type']).toContain('application/json')
+      expect(response.result).toMatchObject({
+        url: expect.any(String)
+      })
```

Comment:
```suggestion
        url: 'https://s3.dummy.com/file.txt'
      })
```

Reply:
Fixed in [c7ee98f](https://github.com/DEFRA/forms-submission-api/pull/8/commits/c7ee98f627c6542a5999c0fbe68362296483ac48), thanks.